ray-operator/controllers/ray/batchscheduler/volcano/volcano_scheduler.go

```diff
@@ -184,9 +182,5 @@ func (vf *VolcanoBatchSchedulerFactory) AddToScheme(scheme *runtime.Scheme) {
 }
 
 func (vf *VolcanoBatchSchedulerFactory) ConfigureReconciler(b *builder.Builder) *builder.Builder {
-	return b.
-		Watches(&source.Kind{Type: &v1beta1.PodGroup{}}, &handler.EnqueueRequestForOwner{
-			IsController: true,
-			OwnerType:    &rayv1.RayCluster{},
-		})
+	return b.Owns(&v1beta1.PodGroup{})
```

Comment:
Note: https://yash-kukreja-98.medium.com/develop-on-kubernetes-series-demystifying-the-for-vs-owns-vs-watches-controller-builders-in-c11ab32a046e